src/main/java/de/srendi/advancedperipherals/common/blocks/PlayerDetectorBlock.java

```diff
@@ -29,17 +28,17 @@ public BlockEntity newBlockEntity(@NotNull BlockPos pos, @NotNull BlockState sta
     }
 
     @Override
-    public @NotNull InteractionResult use(@NotNull BlockState state, Level levelIn, @NotNull BlockPos pos, @NotNull Player player, @NotNull InteractionHand handIn, @NotNull BlockHitResult hit) {
+    public @NotNull InteractionResult useWithoutItem(@NotNull BlockState state, Level levelIn, @NotNull BlockPos pos, @NotNull Player player, @NotNull BlockHitResult hit) {
```

Comment:
are we only detecting right click with an empty hand here? or we may want detect both with and without item

Reply:
Actually not sure, that mojang change confused me a bit
Need to test that

Reply:
Just tested it - also works with item